app/models/ingest.rb

```diff
@@ -35,8 +35,20 @@ def self.find_fields
   scope :created_before, ->(time) { where("created_at < ?", time) unless time.blank? }
   scope :with_bag_id, ->(id) { where(bag_id: id) unless id.blank? }
   scope :with_ingested, ->(ingested) { where(ingested: ingested) if [true,false].include?(ingested) }
-  scope :latest_only, ->(flag) { group(:bag_id).having("created_at = max(created_at)") unless flag.blank? }
-  
+  scope :latest_only, ->(flag) do
+    unless flag.blank?
+      joins("INNER JOIN (
+        SELECT bag_id, MAX(created_at) AS max_created_at
```

Comment:
OK, I was missing the point of the group-by operation and the consequent selection of the latest within each group.